src/Microsoft.Health.Dicom.Functions.UnitTests/Indexing/ReindexDurableFunctionTests.Orchestration.cs

```diff
@@ -31,16 +31,26 @@ public async Task GivenValidTags_WhenAddExtendedQueryTagsOrchestrationAsync_Shou
             IDurableOrchestrationContext context = Substitute.For<IDurableOrchestrationContext>();
             context.GetInput<IReadOnlyList<AddExtendedQueryTagEntry>>().Returns(tagsEntries);
             await _reindexDurableFunction.AddAndReindexTagsAsync(context, NullLogger.Instance);
+
```

Comment:
Can we be consistent with the names use Reindexing everywhere?

Microsoft.Health.Dicom.Functions.UnitTests.Indexing, is called indexing